docs/source/guides/install/docker.rst

```diff
@@ -2,141 +2,92 @@
 Docker
 ======
 
-For automating ``subscriptions.yaml`` downloads to pull new media, see :ref:`this page <guides/getting_started/automating_downloads:docker and unraid>` on how to set up a cron job in any of the docker containers.
```

Comment:
Likewise can be moved to separate PR

Reply:
[Ditto](https://github.com/jmbannon/ytdl-sub/pull/1282#discussion_r2280696754).